UnrealPlugin/README.md

```diff
@@ -47,4 +123,63 @@ public class <projectname>ServerTarget : TargetRules
+```
+
+There are two ways to include the app-local prerequisites - either through the Unreal Engine editor or by editing DefaultGame.ini.
+###### In the Unreal Editor
+In the editor go to Edit -> Project Settings. In the opened window navigate to Packaging on the left side. Scroll to the bottom of the list, and tick "Include app-local prerequisities".
+
+###### In DefaultGame.ini
+Or you can update DefaultGame.ini to show the following:
+ [/Script/UnrealEd.ProjectPackagingSettings]
+IncludeAppLocalPrerequesites=True
+
+If the category already exists in your DefaultGame.ini, then just add the second line to it. This ensures that all app local dependencies ship with the game as well.
+
+Update [game name].Build.cs file to add "PlayfabGSDK" into the PublicDependencyModuleNames.AddRange(); list as follows:
+
+```c_cpp 
+		PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore", "HeadMountedDisplay", "PlayfabGSDK"});
+
+		PrivateDependencyModuleNames.AddRange(new string[] { });
+```
+
+There are two ways to update the Game Instance Class - either through the Unreal Engine editor or by editing DefaultEngine.ini directly. 
+###### In the Unreal Editor
+In the editor, This can also be set through the UI in the editor. In the editor go to Edit -> Project Settings. From that opened window,
+navigate to Maps&Modes on the left side. Scroll to the bottom, and then you can set the option "Game Instance Class" to your new game instance class directly, and avoid typos.
+
+###### In DefaultEngine.ini
+Or you can update DefaultEngine.ini file and add this:
+[/Script/EngineSettings.GameMapsSettings]
+GameInstanceClass=/Script/[game name].MyGameInstance"
+
+Then select the option to build from development editor mode and build the project.
+
+![image depicting Visual Studio with the option to build in Development Editor Mode](Documentation/DevelopmentEditor.png)
+
+Then select the option to build from development server mode and build the project again.
+
+Once the build succeeds, close Visual Studio.
+
+## Packaging the game server
+
+Right click on the .uproject file in your file editor and select "Generate Visual Studio Project Files". 
+
+Then open the .uproject file.
+
+In the top left of the Unreal editor menu, click on file->package->target configuration ->[game name] server and then go to file -> package -> windows x64. 
+You can now use this packaged version of your game server to [test with LocalMultiplayerAgent](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/locally-debugging-game-servers-and-integration-with-playfab) 
+or to use it directly with Playfab by [creating a build](https://developer.playfab.com/) and then using [Playfab MpsAllocatorSample](https://github.com/PlayFab/MpsSamples/blob/master/MpsAllocatorSample/README.md).
+
+To get the game client, click on file->package->target configuration->[game name] client and then go to file -> package -> windows x64. 
```

Comment:
I think we can omit the client connection part and leave it for the sample we're gonna demo on the MpsSamples repository. We can update this Readme with a link once we're done with it!